plugins/nexusoperations/frontend/handler.go

```diff
@@ -91,25 +76,27 @@ type HandlerOptions struct {
 
 type completionHandler struct {
 	HandlerOptions
+	preProcessErrorsCounter metrics.CounterIface
```

Comment:
It doesn't need to be a member of `completionHandler` right? It's only used in `CompleteOperation` method.

Reply:
No good reason to recreate this every request.